packages/lit-dev-content/site/docs/index.md

```diff
@@ -42,7 +42,7 @@ Some things to note:
 * Lit’s [expressive, declarative templates](/docs/templates/overview/) (utilizing JavaScript tagged template literals) make it easy to describe how a component should be rendered.
 * [Reactive properties](/docs/components/properties/) represent a component’s public API and/or internal state; your component automatically re-renders whenever a reactive property changes.
 * [Styles](/docs/components/styles) are scoped by default, keeping your CSS selectors simple and ensuring that your component’s styling won’t pollute (or be polluted by) the surrounding context.
-* This example is in TypeScript, but TypeScript is 100% optional–Lit works great in JavaScript, too.
+* Lit works great in vanilla JavaScript, or you can use TypeScript for more even better ergonomics by using decorators and strong types.
```

Comment:
nit: "for ~more~ even better"

Reply:
But it **is** more even better!

(Fixed.)